koku/koku/settings.py

```diff
@@ -291,6 +291,8 @@
 
 INTERNAL_IPS = ["127.0.0.1"]
 
+DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
```

Comment:
For discussion of this setting, see: https://docs.djangoproject.com/en/4.0/releases/3.2/#customizing-type-of-auto-created-primary-keys

I tried setting this to `BigAutoField` for the reporting app, but the migration didn't go well, so I think we're better off sticking with `AutoField` everywhere.